cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +96,38 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftOrg,
+		"draft-org",
+		"",
+		"",
+		"a Github organization or user where the Release Notes PR will be created",
+	)
+
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftRepo,
+		"draft-repo",
+		"",
+		"",
+		"the name of a Github repository where the Release Notes PR will be created",
```

Comment:
We should mention that this is the fork of the repository

Reply:
I've altered the help message, now it reads:
```
the name of the fork of k/sig-release, the Release Notes Draft PR will be created from this repository
```